src/main/java/org/opentripplanner/routing/api/request/preference/ItineraryFilterPreferences.java

```diff
@@ -31,7 +31,7 @@ public final class ItineraryFilterPreferences {
   private final boolean removeItinerariesWithSameRoutesAndStops;
   private final TransitGeneralizedCostFilterParams transitGeneralizedCostLimit;
   private final CostLinearFunction removeTransitWithHigherCostThanBestOnStreetOnly;
-  private final boolean removeTransitIfWalkingIsBetter;
+  private final boolean filterDirectFlexByEarliestDeparture;
```

Comment:
Is there some other filter thingy for filtering the non-optimal transit based on direct comparison?

Reply:
I don't think I understand the question. Can you please elaborate?

Reply:
The original name of the variable was "removeTransitIfWalkingIsBetter" which made me think this was somehow related to the filter which filters out transit itineraries which are worse than walk only (I guess also can be compared to other street modes). This new name seems to repurpose this completely. So I'm just wondering was the original name completely wrong or did you remove the old and add a new boolean so the diff looks like this?

Reply:
Ah, i get it now. The original variable was completely unused so it now looks like I accidentally replace it but the removed it and put the new one on the same line.

They are completely unrelated.